Light.json

```diff
@@ -3444,6 +3445,7 @@
         "type": "Starship",
         "subType": "Starfighter: X-Wing",
         "uniqueness": "*",
+        "destiny": "0 or 7",
```

Comment:
Please use the high number of `7` as the destiny.
For the low number of `0`, please place in a field of `uniqueDestiny: "0"`